common/test/unit/test_monit_app_configuration.py

```diff
@@ -9,16 +9,10 @@
 
 class TestGodAppInterface(unittest.TestCase):
   def test_create_config_file(self):
-    flexmock(file_io)\
-      .should_receive('write')\
-      .and_return()
-    temp_file = monit_app_configuration.create_config_file("mywatch",
-                                                     "start_cmd",
-                                                     "stop_cmd",
-                                                     [1,2,3],
-                                                     {'ENV1':"VALUE1",
-                                                      'ENV2':"VALUE2"})
-    self.assertIsNone(temp_file)
-    
+    flexmock(file_io).should_receive('write')
+    monit_app_configuration.create_config_file(
+      'mywatch', 'start_cmd', 'pidfile', 4000,
+      {'ENV1': 'VALUE1', 'ENV2': 'VALUE2'})
```

Comment:
Just wanted to check if flexmock really can verify that mocked function was called with specific arguments. It can=) But still I think that interface of mock library is clearer.

```
  def test_create_config_file(self):
    (flexmock(monit_app_configuration)
       .should_receive('find_executable')
       .with_args('bash')
       .and_return('BASH')
    )
    (flexmock(monit_app_configuration)
       .should_receive('find_executable')
       .with_args('start-stop-daemon')
       .and_return('START-STOP')
    )
    (flexmock(monit_app_configuration.file_io)
       .should_receive('write')
       .with_args(
         '/etc/monit/conf.d/appscale-mywatch-4000.cfg',
         'CHECK PROCESS mywatch-4000 PIDFILE pidfile\n'
         '  group mywatch\n'
         '  start program = "START-STOP --start --background --pidfile pidfile '
         '--startas BASH -- -c \'exec env ENV2=VALUE2 ENV1=VALUE1  '
         'start_cmd >> /var/log/appscale/mywatch-4000.log 2>&1\'"\n'
         '  stop program = "BASH -c \'start-stop-daemon --stop '
         '--pidfile pidfile && rm pidfile\'"\n')
       .once()
    )
    monit_app_configuration.create_config_file(
      'mywatch', 'start_cmd', 'pidfile', 4000,
      {'ENV1': 'VALUE1', 'ENV2': 'VALUE2'})
```